docs/source-pytorch/levels/expert_level_24.rst

```diff
@@ -23,14 +23,6 @@ Extend the functionality of the Lightning CLI.
    :height: 150
    :tag: expert
 
-.. displayitem::
-   :header: Run on your own cloud
-   :description: Learn how to run on your Company or University private clouds.
-   :col_css: col-md-6
-   :button_link: ../clouds/run_expert.html
```

Comment:
Based on the description, here, this is probably meant to direct to cluster_intermediate_1.rst?
Or perhaps the content of this page should be the same as the list in clouds/cluster.rst?